lib/trento/application/event_handlers/checks_event_handler.ex

```diff
@@ -16,18 +17,23 @@ defmodule Trento.ChecksEventHandler do
         %ChecksExecutionRequested{cluster_id: cluster_id, hosts: hosts, checks: checks},
         %{correlation_id: execution_id}
       ) do
-    case Checks.request_execution(execution_id, cluster_id, hosts, checks) do
+
+    hosts_settings =
+      Enum.map(hosts, fn host -> Hosts.get_connection_settings(host) end)
```

Comment:
it is  idiomatic to start the pipe with the variable e.g. `hosts |> Enum.map(fn host-> .....`